File: 2630.py
for i in range(0, int(input())):
    conversao = input()
    r, g, b = map(int, input().split(' '))
    match conversao:
        case "min":
            print(f"Caso #{i+1}: {min(r,g,b)}")
        case "max":
            print(f"Caso #{i+1}: {max(r, g, b)}")
        case "mean":
            print(f"Caso #{i+1}: {(r+b+g)//3}")
        case "eye":
            print(f"Caso #{i+1}: {int(0.30*r + 0.59*g + 0.11*b)}")

File: 2630.cpp
#include <iostream>
#include <string>
#include <algorithm>
using namespace std;

int main(){
    int r, g, b, numero_casos;
    string conversao;
    cin >> numero_casos;
    
    for(int i = 0; i < numero_casos; i++){
        cin >> conversao >> r >> g >> b;
        
        int valores[] = {r, g, b};
        
        if(conversao == "min")
            cout << "Caso #" << i+1 << ": " << *min_element(valores, valores + 3) << endl;
        else if(conversao == "max")
            cout << "Caso #" << i+1 << ": " << *max_element(valores, valores + 3) << endl;
        else if(conversao == "mean")
            cout << "Caso #" << i+1 << ": " << (r + g + b) / 3 << endl;
        else
            cout << "Caso #" << i+1 << ": " << (int)(0.30*r + 0.59*g + 0.11*b) << endl;
    }
    
    return 0;
}
